CHANGELOG.txt

```diff
@@ -1,4 +1,4 @@
-1.1.1rc1:
+1.1.1rc1.post0:
```

Comment:
Do we do this now? Might be overkill or not a bad idea. I can't tell ;-)

Reply:
Dunno.. usually we just have branch name in there for dev versions but since release candidates are semi official releases, I put some RC version number last time.. doesnt really matter much I guess